docs/source/tutorial/commercial_paper.md

```diff
@@ -674,8 +676,8 @@ download and their exact versions:
 
 ```json
   "dependencies": {
-    "fabric-network": "~1.4.0",
-    "fabric-client": "~1.4.0",
+    "fabric-network": "^2.1.0",
+    "fabric-ca-client": "^2.1.0",
     "js-yaml": "^3.12.0"
   },
 ```
```

Comment:
I feel we should reword the preceding paragraph and remove the block that has version specific information in it. Since the files sit in `fabric-samples` its highly unlikely this will get updated when the tutorial dependencies are updated.

"Open the `package.json` file to see how `issue.js` identifies the packages to
download and their exact versions by examining the `"dependencies"` section of the file.

Reply:
Thanks, I pushed the suggestion.